### ISSUE TITLE
Temp files generated taking up available server storage

### ISSUE THREAD

**Issue by jopeps** [academic-moodle-cooperation/moodle-local_downloadcenter#27]
There do not appear to be any in built admin features or scheduled tasks to manage the accumulation of temp files generated by the plugin.   Temp files remain within the Moodle appdata/temp directory, taking up available server storage, which can severely impact the site once storage capacity is reached.  

Could admin tools or scheduled tasks be incorporated into the plugin to mange temp files generated?

**Comment by goggo24** [comment]
Which version of the plugin do you use? With v.3.11.1 we implemented download as stream zip.

**Comment by jopeps** [comment]
Hi,  Thank you for the update.  We are using version v3.9.1.  Are admin tools and or scheduled tasks included in version v.3.11.1, to manage the temp files generated?

**Comment by goggo24** [comment]
No, they are not included - as we hadn't have any troubles until now, we didn't implemented anything in this direction. I will take the topic in our backlog and discuss it within the team.

**Comment by mwreichel** [comment]
Hi,
since we have changed the way how files are downloaded (with version 3.11.1), no temp files are generated anymore.
The issue will be closed.

All the best,
Martin
